moveit_setup_assistant/templates/moveit_config_pkg_template/launch/run_benchmark_ompl.launch

```diff
@@ -15,7 +15,6 @@
 
   <!-- Start Benchmark Executable -->
   <node name="$(anon moveit_benchmark)" pkg="moveit_ros_benchmarks" type="moveit_run_benchmark" args="$(arg cfg) --benchmark-planners" respawn="false" output="screen">
-    <rosparam command="load" file="$(find [GENERATED_PACKAGE_NAME])/config/kinematics.yaml"/>
```

Comment:
Same here. Did you test that these apps still find the kinematics parameters?

Reply:
Yes, even though the tutorial was somewhat hard to follow ... which I hope to address at some point